### ISSUE TITLE
Android emulators fail to start in the last day

### ISSUE THREAD

**Issue by sjudd** [travis-ci/travis-ci#8946]
Android API 16, 17, and 18 emulators hang forever and fail to start, here's an example: https://travis-ci.org/bumptech/glide/jobs/318500648. 

To verify it's not due to any change I've made, I re-ran the last successful build and I see the same failures on the same emulators: https://travis-ci.org/bumptech/glide/jobs/317932777.

Did anything change on the travis side in the last day or so that might be contributing to this?

**Comment by sjudd** [comment]
Precise seems to do a bit better than Trusty, though the API 16 emulator still fails to start: https://travis-ci.org/sjudd/glide/builds/320039560

**Comment by sjudd** [comment]
I believe this is due to the 27.0.2, which was released some time in December. My travis script includes sdkmanager --update since travis seems to default to a relatively old version of the Android toolchain. I'd guess that the failure started when that update started installing 27.0.2 of the emulator.

Since I'm able to reproduce the failure locally, I don't think this has anything to do with travis-ci.

In particular I was able to fix the issue by removing ``--device "Nexus 5X"`` from the ``avdmanager create avd`` line. Either using ``echo no | avdmanager create avd <...>`` or using ``--device "Nexus 4"`` seem to work. 

In case it's helpful for anyone else, I'll post further updates here: https://github.com/bumptech/glide/issues/2738
